srcpkgs/efibootmgr/files/README.voidlinux

```diff
@@ -2,8 +2,8 @@ efibootmgr can automatically generate EFI boot entries to
 directly boot the kernel after every kernel update or installation.
 Enable this by editing /etc/default/efibootmgr-kernel-hook.
 
-The EFI variables need to be accessible in order for this to work, so
-add this line to your fstab and reboot or mount manually:
+The EFI variables need to be accessible in order for this to work, either
+boot into EFI mode or add this line to your fstab and reboot or mount:
```

Comment:
Booting via UEFI isn't sufficient to expose `efivarfs`. It must be manually mounted or `elogind` must run to mount it. (I have no idea why `elogind` cares about `efivarfs`, but it will automatically mount it when seeting up other mounts.)

Reply:
This part has now been reworded.